pkg/controller/v1beta1/inferenceservice/reconcilers/deployment/deployment_reconciler.go

```diff
@@ -87,13 +87,18 @@ func createRawDeploymentODH(clientset kubernetes.Interface, componentMeta metav1
 	if err != nil {
 		return nil, err
 	}
-	if val, ok := componentMeta.Labels[constants.ODHKserveRawAuth]; ok && val == "true" {
-		for _, deployment := range deploymentList {
+	for _, deployment := range deploymentList {
```

Comment:
IIRC, the `deploymentList` is for supporting multi-node. It only makes sense to mount TLS Keys to the head node. So, I'm not seeing OK to modify all deployments in the list. We may also want to _fix_ injecting oauth-proxy also only on the head node.

That said, IMO in this PR we should **not** be impacting InferenceServices. None of the runtimes are well prepared for consuming the mounted TLS certs. For this, more work is needed. So, I'd suggest to revert changes to ISVCs from this PR and deal only with InferenceGraph. The changes for ISVCs require more analysis.